### ISSUE TITLE
The trouble when using explore_model.ipynb of Akita

### ISSUE THREAD

**Issue by gxy-huangdi** [calico/basenji#120]
1. About params.json, I meet a problem:
   
<img width="664" alt="image" src="https://user-images.githubusercontent.com/57659163/175315979-13c4b2da-c029-4ce6-80f2-6f54c6bf3fe6.png">
This figure show that the number of parameters is different from yours. And it seems to lack of batch_normalization.

2. About result figure:
   
<img width="1440" alt="image" src="https://user-images.githubusercontent.com/57659163/175317068-75d4c8b3-b032-44a7-8bab-ef2c549f010f.png">
I wonder if this is due to missing parameters.
Hope to get your answer!

**Comment by davek44** [comment]
Hi, I recently made a change to how batch normalization is communicated via the parameters file. Pull the latest code from github and make sure your parameters file has this line: https://github.com/calico/basenji/blob/master/manuscripts/akita/params.json#L19

**Comment by gxy-huangdi** [comment]
> Hi, I recently made a change to how batch normalization is communicated via the parameters file. Pull the latest code from github and make sure your parameters file has this line: https://github.com/calico/basenji/blob/master/manuscripts/akita/params.json#L19

Thank you very much. I have solved this problem. In addition, I have one final question: can the Akita model be used directly to predict chromosome interactions in nerve cells? How so I don't have to train on my own data.
